apps/wallet/tests/utils/getLocalTokens.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Page } from '@playwright/test';
+
+import { expect } from '../fixtures';
+
+export async function getLocalTokens(page: Page) {
+    await page.getByTestId('menu').click();
```

Comment:
We actually should set this at the build level so that we don't need to drive the UI for this. It should just be the default.

Reply:
This is already set in `e2e.yml` so I just modified the existing tests to not do the network selection